packages/cxl-ui/src/components/cxl-save-playbooks.js

```diff
@@ -0,0 +1,94 @@
+import '@conversionxl/cxl-lumo-styles';
+import { customElement, html, LitElement, property, query } from 'lit-element';
+import cxlSavePlaybooksStyles from '../styles/cxl-save-playbooks-css';
+
+@customElement('cxl-save-playbooks')
+export class CXLSavePlaybooksElement extends LitElement {
+  @query('a')
+  anchor;
+
+  @query('a > iron-icon')
+  ironIcon;
+
+  /**
+   * Is playbook currently in the "saved" state
+   */
+  @property({ type: Boolean, reflect: true })
+  selected;
+
+  /**
+   * if true, text "Save / Unsave Playbook" won't be displayed
+   */
+  @property({ type: Boolean, reflect: true })
+  starOnly;
+
+  @property({ type: Number, reflect: true })
+  playbookId;
+
+  @property({ type: Number, reflect: true })
+  userId;
+
+  /**
+   * API Url to which we make a POST request
+   *
+   * @type {string}
+   */
+  @property({ type: String, reflect: true })
+  apiUrl = 'https://jsonplaceholder.typicode.com/posts';
+
+  static get styles() {
+    return [cxlSavePlaybooksStyles];
+  }
+
+  // eslint-disable-next-line class-methods-use-this
+  async postData(url = '', data = {}) {
+    const response = await fetch(url, {
+      method: 'POST',
+      mode: 'cors',
+      cache: 'no-cache',
+      credentials: 'same-origin',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      redirect: 'follow',
+      referrerPolicy: 'no-referrer',
+      body: JSON.stringify(data),
+    });
+
+    return response.status;
+  }
+
+  async sendToApi() {
+    return this.postData(this.apiUrl, {
+      playbookId: this.playbookId,
+      userId: this.userId,
+      save: this.selected,
+    });
+  }
+
+  firstUpdated(_changedProperties) {
+    super.firstUpdated(_changedProperties);
+
+    this.anchor.addEventListener('click', async (e) => {
```

Comment:
Rather than use the programmatic way of registering a listener, why not use the `@click` syntax?

Reply:
That's a preference issue I think. I'm always a little bit confused looking at the other components in the UI, because in many places it's not really consistent. Examples:

https://github.com/conversionxl/aybolit/blob/master/packages/cxl-ui/src/components/cxl-marketing-nav.js#L57
https://github.com/conversionxl/aybolit/blob/master/packages/cxl-ui/src/components/cxl-marketing-nav.js#L95
https://github.com/conversionxl/aybolit/blob/master/packages/cxl-ui/src/components/cxl-marketing-nav.js#L144
https://github.com/conversionxl/aybolit/blob/master/packages/cxl-ui/src/components/cxl-marketing-nav.js#L162

@anoblet can you please elaborate regarding the reason when to use and not-use `@click` syntax and should the above example be refactored to `@click`?

Reply:
It's typically cleaner, and requires less code. `cxl-marketing-nav` was created before I joined the team. If at all possible, I would use the `@click` syntax.

Reply:
@anoblet thanks for the insight, I'll try to stick to the `@click` whenever possible